mlflow/models/model.py

```diff
@@ -157,6 +159,16 @@ def mlflow_version(self):
         """
         return self._mlflow_version
 
+    @property
+    def metadata(self) -> Optional[Dict[str, Any]]:
+        """
+        .. Note::
+            Experimental: This method may change or be removed in a future release without warning.
+
+        User defined metadata added to the model
```

Comment:
Can we add an example within the doc string here that shows setting a dictionary of values to a model? For reference, see `models.add_libraries_to_model()`. This same example can be applied (with needed modifications) beneath lines 246 as well.